### PR TITLE
TSERV-1006 Exit on fail after writing report.

### PR DESCRIPTION
**Pull request by lootic**

### CODE REVIEW

File: bin/run_functions.js [hunk 1]
```diff
@@ -494,6 +491,10 @@ function executeProject(filename, project, options) {
                                 jobs.reportForTestJob(jobId, options['output'], options['reportFileName'], 'format' in options ? options['format'] : "junit");
                             }
                         }
+
+                        if(status === 'FAILED') {
+                            process.exitCode = 1;
```
Comment (bsubba) [inline bin/run_functions.js:496]: how/where  will this be used?

Comment (lootic) [reply bin/run_functions.js:496]: https://stackoverflow.com/questions/5266152/how-to-exit-in-node-js

See second answer. This way we set it for use "later", so as long we don't do exit(0) or set this variable to some other value  we will exit when we are done with the other things but with this status code.

Comment (akkikumar72) [reply bin/run_functions.js:496]: I guess just `process.exitCode(1)` will work the same way

**Review by sverdlovka — APPROVED** [review]

File: bin/run_functions.js (hunk 1 above)
Comment (lootic) [reply bin/run_functions.js:496]: exitCode is not a function, it's a property and process.exit(1) would shut down the process straight away which we don't want.

**Review by akkikumar72 — APPROVED** [review]

**Pull request merged**